src/AwsServiceProvider.php

```diff
@@ -37,30 +44,39 @@ class AwsServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        $this->app['aws'] = $this->app->share(function ($app) {
-            // Retrieve the config
-            $config = $app['config']['aws'] ?: $app['config']['aws::config'];
-            if (isset($config['config_file'])) {
-                $config = $config['config_file'];
-            }
+        $this->app['aws'] = $this->app->share(function ($app)
```

Comment:
Shouldn't we use `$this->app->singleton('aws', function($app) {...});` like in the [docs](http://laravel.com/docs/master/providers#basic-provider-example)?